lib/improver/nbhood.py

```diff
@@ -170,6 +172,36 @@ def get_neighbourhood_width_in_grid_cells(
             )
         return grid_cells_x, grid_cells_y
 
+    @staticmethod
+    def adjust_nsize_for_ens(ens_factor, num_ens, width_in_km):
+        """
+        Adjust neighbourhood size according to ensemble size.
+
+        Parameters
+        ----------
+        ens_factor : float
+            Defined Factor with which to adjust the neighbour size
+            If ens_factor = 1.0 this essentially conservers ensemble
+            members if every grid square is considered to be the
+            equivalent of an ensemble member.
+        num_ens : float
+            Number of realizations / ensemble members.
```

Comment:
There are lots of places in this code where / has been used instead of or e.g. x/y and time/forecast_reference_time. I've replaced them where I can but some would require modifications to the unit tests in code I haven't written so I've left those in. If you feel strongly about these please can you raise technical debt ticket to have these corrected.